sagol/run_models.py

```diff
@@ -47,6 +47,13 @@ def get_or_create_models(experiment_data: ExperimentData, task_name: str, ylabel
     pre_computed_models = get_pre_computed_models(ylabel, roi_paths, experiment_data.shape)
     return pre_computed_models or generate_models(masked_experiment_data, task_name, ylabel, roi_paths, model_params)
 
+def get_or_create_models_for_all_tasks_together(experiment_data: ExperimentData, ylabel: str,
```

Comment:
I think we should remove code duplication and merge this into `get_or_create_models`, making the `task_name` there be:
1. A list if we want multiple specific tasks.
2. None if we want all tasks.